runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessDetailsDialog/ProcessDetailsTabs/SchedulesTab.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect } from 'react';
+
+import { Box, Chip, Typography } from '@mui/material';
+import cronstrue from 'cronstrue/i18n';
+import i18n from 'i18next';
+import moment from 'moment';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import { useDispatch, useSelector } from '#src-app/store';
+import {
+    scheduleProcessActions,
+    scheduleProcessSelector,
+} from '#src-app/store/slices/ScheduleProcess';
+
+import {
+    DetailsInfoTabProps,
+    ProcessDetailsTab,
+} from '../ProcessDetailsDialog.types';
+
+export const SchedulesTab = ({ value, process }: DetailsInfoTabProps) => {
+    const { translate } = useTranslations();
+    const dispatch = useDispatch();
+    const { schedules } = useSelector(scheduleProcessSelector);
+    const processId = process?.id;
+    const lastRun = process.lastRun
+        ? moment(process.lastRun).fromNow()
```

Comment:
moment is outdated, if possible use https://date-fns.org/ recently added by wiktor pilak